storage/src/adb/any/fixed/sync.rs

```diff
@@ -1,6 +1,6 @@
 use crate::{
     // TODO(https://github.com/commonwarexyz/monorepo/issues/1873): support any::fixed::ordered
-    adb::{self, any::fixed::unordered::Any, operation::Fixed},
+    adb::{self, any::fixed::unordered::Any, operation::fixed::unordered::Operation},
```

Comment:
nit: for consistency with tests should we do `as Fixed` here?

Reply:
I've been trying to consistently just use "Operation" as I think that's more clear in code that is only working with one variant. Did I miss renaming this in some tests? (Will take a look...)

Reply:
fixed the remaining one in tests from this file.